storage/mysql/kubernetes/resetdb.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Pick a random port between 2000 - 34767. Will fail if unlucky and the port is in use.
+readonly port=$((${RANDOM} + 2000))
+kubectl port-forward "galera-0" "${port}:3306" >/dev/null &
+proxy_pid=$!
+
+options_file=$(mktemp)
+chmod 0600 "${options_file}"
+echo > ${options_file} <<EOF
+[client]
+host=localhost
+port=${port}
+user=root
+password="$(kubectl get secrets mysql-credentials --template '{{index .data "root-password"}}' | base64 -d)"
+EOF
+
+"${GOPATH}/src/github.com/google/trillian/scripts/resetdb.sh" \
```

Comment:
All the stuff that relies on gopath will presumably break if there's more than one entry in it. not sure this matters.